packages/site/src/content/DataTable/DataTableNotes.mdx

```diff
@@ -11,8 +13,15 @@ with libraries like [TanStack Table](https://tanstack.com/table) to handle
 complex data management.
 
 To see examples of how the atomic DataTable components can be configured, see
-[this section of storybook](../?path=/story/components-lists-and-tables-datatable-web-composable--basic).
-These are just starting points, the components are fully composable and can
+
+<a
+  href={getStorybookUrl(
+    "?path=/story/components-lists-and-tables-datatable-web-composable--basic"
+  )}
+>
+  this section of storybook
+</a>
+. These are just starting points, the components are fully composable and can
```

Comment:
ahhhh linter formatting them like this on commit I think. They get the job done though with `getStorybookUrl`